pkg/alertmanager/amcfg.go

```diff
@@ -1718,6 +1724,22 @@ func (cb *ConfigBuilder) convertProxyConfig(ctx context.Context, in monitoringv1
 	return out, nil
 }
 
+func (cb *ConfigBuilder) convertGlobalJiraConfig(out *globalConfig, in monitoringv1.GlobalJiraConfig) error {
```

Comment:
(suggestion) I'd pass a pointer and check for nil first so we don't have to do it in convertGlobalConfig().
```suggestion
func (cb *ConfigBuilder) convertGlobalJiraConfig(out *globalConfig, in *monitoringv1.GlobalJiraConfig) error {
    if in == nil {
        return nil
    }
```